tests/test_csv2pq.py

```diff
@@ -0,0 +1,183 @@
+
+    inFiles = ["data/testfile1.csv", "data/testfile2.csv"]
+    inSchem = "data/test.schema"
+    outFile = ["/tmp/testfile1.pq", "/tmp/testfile2.pq"]
+    numTest = 1
+
+
+# *****************************************************************************
+# *                              c h k 4 F i l e                              *
+# *****************************************************************************
+
+def chk4File(flist, rm=False):
+    "Verify that created output files actually exist."
+
+    for file in flist:
+        if not os.path.exists(file):
+            print('Output file "' + file + '" not found.')
+            return False
+        if (rm): os.unlink(file)
+    return True
+
+
+# *****************************************************************************
+# *                               r u n T e s t                               *
+# *****************************************************************************
+
+def runTest(cmd, indata=None):
+    "Run a command to test it."
+
+    nTest = str(TestInfo.numTest)
+    TestInfo.numTest += 1
+    print("Running test", nTest + ':', cmd)
+
+    # We would use the newfangled subprocess options to do this but those
+    # are broken in various versions of python 3 so are undependable.
+    #
+    if indata:
+        rPipe, wPipe = os.pipe()
+        os.write(wPipe, bytes(indata, 'ascii'))
+        os.close(wPipe)
+    else:
+        rPipe = None
+
+    try:
+        status = subprocess.run(cmd.split(), stdin=rPipe)
+    except Exception as err:
+        print('Test', nTest, 'ended with an exception;', err)
+        return None
+
+    print("Test", nTest, 'ended with RC =', status.returncode)
+    print('-' * 79)
+    return status.returncode == 0
+
+
+# *****************************************************************************
+# *                          s e t u p _ m o d u l e                          *
+# *****************************************************************************
+
+def setup_module():
+    "Setup environment for testing."
+
+    # Make sure all the files we need are present
+    #
+    for file in TestInfo.inFiles + [TestInfo.inSchem]:
+        if not os.path.exists(file):
+            raise Exception('Required file "' + file + '" not found.')
+        if not os.path.isfile(file):
+            raise Exception('Required file "' + file + '" is not a file.')
+
+    # Start with a clean slate
+    #
+    for file in TestInfo.outFile:
+        try:
+            os.unlink(file)
+        except Exception:
+            pass
+
+    # Get the current working directory for subsequent envar settings
+    #
+    curDir = os.path.dirname(os.path.abspath(__file__))
+
+    # Augment python path for fidning modules
+    #
+    if 'PYTHONPATH' in os.environ:
+        oldEnv = os.environ['PYTHONPATH']
+    else:
+        oldEnv = ''
+    os.environ['PYTHONPATH'] = curDir + '/../python/parquet_tools:' + oldEnv
```

Comment:
Well, I couldn't get the unittest (or pytest) to set that path properly. I'm open to suggestions but the way the directories are laid out you simply can't run a test out of a git repo.

Reply:
If you are using pure setuptools you probably want to be using `python setup.py develop` so that things will work without manually hacking PYTHONPATH.

Reply:
I guess you are supposed to have PYTHONPATH/PATH setup already when you run unit tests? This I think depends on how do you setup your development environment. If you chose not to use eups then you should do what people normally do when they do distutisl-based development.